hack/faq.repo

```diff
@@ -0,0 +1,10 @@
+[ecnahc515-faq]
+name=Copr repo for faq owned by ecnahc515
+baseurl=https://copr-be.cloud.fedoraproject.org/results/ecnahc515/faq/epel-7-$basearch/
+type=rpm-md
+skip_if_unavailable=True
```

Comment:
we probably want this to fail?

Reply:
If faq can't be installed, it will fail. This is the file from copr, so I didn't really touch it. Could probably change that still.

Reply:
I just like the idea of things failing early and failing clearly -- wouldn't be *hard* to find why it failed to install, but it'd be even easier if it failed to try to install. I won't block on this, your decision

Reply:
Fixed

Reply:
Fixed